lib/matplotlib/transforms.py

```diff
@@ -2685,6 +2685,26 @@ def get_matrix(self):
         return self._mtx
 
 
+class ScaledRotation(Affine2DBase):
+    """
+    A transformation that applies offset and direction
+    based on *trans_shift*.
```

Comment:
There's no offset and direction here:
```suggestion
    A transformation that applies rotation by *theta*, after transform by *trans_shift*.
```